### PR TITLE
Cloudformation yaml

### PR DESCRIPTION
**Pull request by tomrf1**
1. cloudformation json -> yaml
2. Reduce JVM memory allocation. It was too big for the instance size, which meant the app has been crashing with an OOM every few hours for...quite some time

### CODE REVIEW

File: cloudformation.yaml [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+            adduser --disabled-password content-api
+            aws configure set region ${AWS::Region}
+            aws s3 cp s3://content-api-dist/authorized_keys /home/ubuntu/.ssh/authorized_keys
+
+            cd /home/content-api
+            mkdir logs
+            mkdir -p /etc/gu
+            aws s3 cp s3://content-api-config/podcasts-rss/${Stage}/podcasts-rss.conf /etc/gu/podcasts-rss.conf
+            aws s3 cp s3://content-api-dist/${Stack}/${Stage}/podcasts-rss/podcasts-rss.service /etc/systemd/system/podcasts-rss.service
+            aws s3 cp s3://content-api-dist/${Stack}/${Stage}/podcasts-rss/podcasts-rss-0.1-SNAPSHOT.tgz .
```
Comment (regiskuckaertz) [inline cloudformation.yaml]: Just curious: is there a change this version changes?

Comment (tomrf1) [inline cloudformation.yaml]: No, not unless we change it in the build file

**Comment by oilnam** [comment]
> It was too big for the instance size, which meant the app has been crashing with an OOM every few hours

Ha, good spot! How did you find out?

**Comment by tomrf1** [comment]
@oilnam I checked the logs before doing the cloudformation change and noticed it had OOM'd. Then saw it had been repeatedly trying to use more memory than a t2.micro has

**Pull request merged**